pkgs/by-name/ki/kind/package.nix

```diff
@@ -10,21 +10,21 @@
 
 buildGoModule rec {
   pname = "kind";
-  version = "0.27.0";
+  version = "0.28.0";
```

Comment:
0.28.0 updated to containerd v2.1.0, and it is reported with CVE-2025-47290

https://github.com/containerd/containerd/security/advisories/GHSA-cm76-qm8v-3j95

https://github.com/kubernetes-sigs/kind/compare/v0.27.0...v0.28.0#diff-55b511fe27bbfb1625d543b65b6b81e030b8ca93b9437b8b48666344ae9d162dR125

Released https://github.com/kubernetes-sigs/kind/releases/tag/v0.29.0 which applied the patch